proxy/logging/Log.cc

```diff
@@ -925,6 +925,7 @@ Log::init(int flags)
     if (config_flags & STANDALONE_COLLATOR) {
       config->collation_mode = Log::COLLATION_HOST;
     }
+    init_fields();
```

Comment:
Both sided of this branch now call ``init_fields()``. Do you want to hoist the call up and simplify?